tensorflow/lite/g3doc/guide/build_rpi.md

```diff
@@ -5,87 +5,89 @@ Raspberry Pi. If you just want to start using TensorFlow Lite to execute your
 
-To cross compile TensorFlow Lite, first install the toolchain and libs:
+To cross compile TensorFlow Lite follow the steps:
 
-```bash
-sudo apt-get update
-sudo apt-get install crossbuild-essential-armhf
-# The following is only needed for Pi Zero build.
-sudo apt-get install crossbuild-essential-armel
-```
+1. Clone official Raspberry Pi cross-compilation toolchain:
 
-If you are using Docker, you may not use `sudo`.
+    ```bash
+    git clone --depth 1 https://github.com/raspberrypi/tools.git rpi_tools
+    ```
 
-Now git-clone the TensorFlow repository
-(`https://github.com/tensorflow/tensorflow`)—if you're using the TensorFlow
-Docker image, the repo is already provided in `/tensorflow_src/`—and then run
-this script at the root of the TensorFlow repository to download all the
-build dependencies:
+2. Clone TensorFlow repository:
+
+    ```bash
+    git clone --depth 1 https://github.com/tensorflow/tensorflow.git tensorflow_src
+
+    ```
 
-```bash
-./tensorflow/lite/tools/make/download_dependencies.sh
-```
+    **Note:** If you're using the TensorFlow Docker image, the repo is already provided in `/tensorflow_src/`.
 
-Note that you only need to do this once.
+3. Run following script at the root of the TensorFlow repository to download all the
+build dependencies:
+
+    ```bash
+    cd tensor_src && ./tensorflow/lite/tools/make/download_dependencies.sh
+    ```
 
-You should then be able to compile:
+    **Note:** You only need to do this once.
 
-To build ARMv7 binary for Raspberry Pi 2, 3 and 4:
+4. To build ARMv7 binary for Raspberry Pi 2, 3 and 4 execute:
 
-```bash
-./tensorflow/lite/tools/make/build_rpi_lib.sh
-```
+    ```bash
+    PATH=$PATH:../rpi_tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/ ./tensorflow/lite/tools/make/build_rpi_lib.sh
+    ```
 
-This should compile a static library in:
-`tensorflow/lite/tools/make/gen/rpi_armv7l/lib/libtensorflow-lite.a`.
+    **Note:** This should compile a static library in:
+    `tensorflow/lite/tools/make/gen/rpi_armv7l/lib/libtensorflow-lite.a`.
 
-To build ARMv6 binary for Raspberry Pi Zero:
+5. To build ARMv6 binary for Raspberry Pi Zero execute:
 
-```bash
-./tensorflow/lite/tools/make/build_rpi_lib.sh TARGET_ARCH=armv6
-```
+    ```bash
+    PATH=$PATH:../rpi_tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/ ./tensorflow/lite/tools/make/build_rpi_lib.sh TARGET_ARCH=armv6
+    ```
 
-This should compile a static library in:
-`tensorflow/lite/tools/make/gen/rpi_armv6/lib/libtensorflow-lite.a`.
+    **Note:** This should compile a static library in:
+    `tensorflow/lite/tools/make/gen/rpi_armv6/lib/libtensorflow-lite.a`.
 
 ## Compile natively on Raspberry Pi
 
-This has been tested on Raspberry Pi 3b, Raspbian GNU/Linux 9.1 (stretch), gcc version 6.3.0 20170516 (Raspbian 6.3.0-18+rpi1).
+Instruction has been tested on Raspberry Pi 3b, Raspbian GNU/Linux 9.1 (stretch), gcc version 6.3.0 20170516 (Raspbian 6.3.0-18+rpi1):
```

Comment:
Is the gcc version the same? "gcc version 6.3.0 20170516"

Reply:
fixed